### PR TITLE
Fix 387: TravisCI is failing

### PR DESCRIPTION
**Pull request by odanado**
ref: #387

### CODE REVIEW

**Comment by odanado** [comment]
Occurred an error.

```
(node:6761) UnhandledPromiseRejectionWarning: /home/travis/build/GoogleChrome/rendertron/middleware/node_modules/ava/lib/serialize-error.js:39
		} catch {}
		        ^
SyntaxError: Unexpected token {
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:617:28)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/travis/build/GoogleChrome/rendertron/middleware/node_modules/ava/lib/api.js:18:24)
(node:6761) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:6761) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by odanado** [comment]
`ava 3.0.0` no longer supports nodejs v8. Therefore, the above error has occurred.

https://github.com/avajs/ava/releases/tag/v3.0.0
> Supported Node.js versions
> We now support Node.js 10, 12 and 13. The minimal versions are 10.18.0, 12.14.0 and 13.5.0 respectively.

**Comment by odanado** [comment]
Do you want to update the nodejs version of Travis CI from 8 to 12?

**Comment by AVGP** [comment]
Yeah, let's do that. The relevant line is in [package.json](https://github.com/GoogleChrome/rendertron/blob/master/package.json#L8), please include this in the PR :)

File: src/filesystem-cache.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ export class FilesystemCache {
     this.cacheConfig = this.config.cacheConfig;
 
     if (!fs.existsSync(this.getDir(''))) {
-      fs.mkdirSync(this.getDir(''));
+      fs.mkdirSync(this.getDir(''), { recursive: true });
```
Comment (odanado) [inline src/filesystem-cache.ts:52]: `src/filesystem-cache.ts` is failing. Because `fs.mkdirSync` does not create a directory recursively by default.

**Comment by odanado** [comment]
`datastore-cache-test › refreshCache refreshes cache` is failing 🤔
https://travis-ci.org/github/GoogleChrome/rendertron/jobs/683940122

I will try to find out the cause of the error.

**Comment by git-ekuo** [comment]
@odanado , @AVGP 

This issue: https://travis-ci.org/github/GoogleChrome/rendertron/jobs/683940122 that you are experiencing shall be addressed by https://github.com/GoogleChrome/rendertron/issues/381. I already have a PR open for it, and it's failing with the issue you are trying to fix. I'd suggest that we let this one go first without fixing the refreshCache test error.

What do you think?

**Comment by AVGP** [comment]
Thanks @odanado for this! Next up is looking into the failing cache test. That one used to be flaky and it's being looked into in #381 :)

**Pull request merged**